snap/local/helper-go/configure.go

```diff
@@ -17,15 +17,15 @@
 package main
 
 import (
-	"strings"
-
 	"github.com/canonical/edgex-snap-hooks/v2/log"
 	"github.com/canonical/edgex-snap-hooks/v2/options"
 	"github.com/canonical/edgex-snap-hooks/v2/snapctl"
 )
 
 // configure is called by the main function
 func configure() {
+	const service = "device-virtual"
+
```

Comment:
It's true that the EdgeX service name is "device-virtual", however here you are setting and passing the app name used in the snap. The snap "service" name is actually "edgex-device-virtual.device-virtual" which is the snap name followed by the app name.